### PR TITLE
Add EdDSA branch for switch statement to handle EdDSA signature packet.

### PR DESCRIPTION
**Pull request by iseki-masaya**
This PR is for https://github.com/open-keychain/open-keychain/issues/1279.

I also erase white spaces.

I test the code following way:

```
$ cd open-keychain/extern/spongycastle/
$ git remote add iskmsy https://github.com/iseki-masaya/spongycastle.git
$ git fetch iskmsy
$ git checkout -b iskmsy/1279
$ cd ../../
$ ./gradlew build
```

I think we have to add import test.

### CODE REVIEW

File: pg/src/main/java/org/spongycastle/bcpg/SignaturePacket.java [hunk 1]
```diff
@@ -26,17 +26,17 @@
     private SignatureSubpacket[]   hashedData;
     private SignatureSubpacket[]   unhashedData;
     private byte[]                 signatureEncoding;
-    
+
```
Comment (dschuermann) [inline pg/src/main/java/org/spongycastle/bcpg/SignaturePacket.java]: Please remove these indent changes from the pull request. Especially when we later want to get this upstream to BouncyCastle, they like to have clean patch sets without unnecessary changes.

**Comment by iseki-masaya** [comment]
I remove indent changes and merge the commit.

File: pg/src/main/java/org/spongycastle/bcpg/SignaturePacket.java [hunk 2]
```diff
@@ -162,6 +162,18 @@ else if (p instanceof SignatureCreationTime)
             signature[0] = ecR;
             signature[1] = ecS;
             break;
+        case EDDSA:
+            {
```
Comment (dschuermann) [inline pg/src/main/java/org/spongycastle/bcpg/SignaturePacket.java:166]: following their style, there shouldn't be brackets here.

**Comment by dschuermann** [comment]
Superceeded by https://github.com/open-keychain/spongycastle/pull/4

**Pull request closed without merging**